### PR TITLE
removed PoolNominationRevoked event from PoolModule

### PR DESCRIPTION
**Pull request by fritzschoff**

### CODE REVIEW

File: packages/synthetix-main/contracts/modules/core/PoolModule.sol [hunk 1]
```diff
@@ -67,6 +67,18 @@ contract PoolModule is IPoolModule {
...
+        Pool.onlyPoolOwner(poolId, msg.sender);
+
+        Pool.Data storage pool = Pool.load(poolId);
+
+        address nominatedOwner = pool.nominatedOwner;
+
+        Pool.load(poolId).nominatedOwner = address(0);
+
+        emit PoolNominationRevoked(poolId, nominatedOwner);
+    }
```
Comment (Rickk137) [inline packages/synthetix-main/contracts/modules/core/PoolModule.sol:80]: `revokePoolNomination` is redundant as there is a similar function already:
`function renouncePoolOwnership(uint128 poolId) external override {
        Pool.onlyPoolOwner(poolId, msg.sender);

        Pool.load(poolId).nominatedOwner = address(0);

        emit PoolOwnershipRenounced(poolId, msg.sender);
    }`

**Review by Rickk137 — CHANGES_REQUESTED** [review]

File: packages/synthetix-main/contracts/modules/core/PoolModule.sol (hunk 1 above)
Comment (fritzschoff) [reply packages/synthetix-main/contracts/modules/core/PoolModule.sol:80]: hmm you are right. Maybe we should get rid of the event?

Comment (Rickk137) [reply packages/synthetix-main/contracts/modules/core/PoolModule.sol]: @noahlitvin do we want pool nominated owners be able to renounce their nomination?

Comment (fritzschoff) [reply packages/synthetix-main/contracts/modules/core/PoolModule.sol]: hold on, I'm I stupid or does this func `renouncePoolOwnership` do the same than `renouncePoolNomination `

**Review by Rickk137 — CHANGES_REQUESTED** [review]

File: packages/hardhat-router/test/fixture-projects/custom-proxy/contracts/Proxy.sol [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/proxy/UUPSProxy.sol";
+
+// solhint-disable-next-line no-empty-blocks
+contract Proxy is UUPSProxy(0x0000000000000000000000000000000000000001) {
+
```
[the remark this answers is not in the document]
Comment (fritzschoff) [reply packages/hardhat-router/test/fixture-projects/custom-proxy/contracts/Proxy.sol:8]: removed it

File: packages/synthetix-main/contracts/interfaces/IPoolModule.sol [hunk 3]
```diff
@@ -8,9 +8,8 @@ interface IPoolModule {
     event PoolCreated(uint128 indexed poolId, address indexed owner);
     event NominatedPoolOwner(uint128 indexed poolId, address indexed owner);
     event PoolOwnershipAccepted(uint128 indexed poolId, address indexed owner);
-    event PoolNominationRenounced(uint128 indexed poolId, address indexed owner);
     event PoolNominationRevoked(uint128 indexed poolId, address indexed owner);
-    event PoolOwnershipRenounced(uint128 indexed poolId, address indexed owner);
+    event PoolOwnerNominationRenounced(uint128 indexed poolId, address indexed owner);
```
Comment (Rickk137) [inline packages/synthetix-main/contracts/interfaces/IPoolModule.sol:12]: let's keep these two ``PoolNominationRevoked``, ``PoolOwnerNominationRenounced`` consistant

**Review by Rickk137 — CHANGES_REQUESTED** [review]

**Review by Rickk137 — APPROVED** [review]

**Pull request merged**